CHANGELOG.md

```diff
@@ -35,7 +35,6 @@
 
 ### Improvements
 
-- Escape queue names properly so they work with HPA ([#1168](https://github.com/kedacore/keda/issues/1168))
```

Comment:
I would consider still leaving the issue here, since it has been fixed :) Up to you however :)